<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin"><number>14</number></property>
   <property name="topMargin"><number>14</number></property>
   <property name="rightMargin"><number>14</number></property>
   <property name="bottomMargin"><number>14</number></property>

   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">
QFrame{background:#171a21;border-radius:16px;}
QLabel{color:white;}
QPushButton{height:56px;border-radius:14px;color:white;background:#222634;font-size:16px;padding:10px;}
QPushButton:pressed{background:#2d3245;}
      </string>
     </property>
     <layout class="QVBoxLayout" name="layoutInner">
      <property name="leftMargin"><number>20</number></property>
      <property name="topMargin"><number>20</number></property>
      <property name="rightMargin"><number>20</number></property>
      <property name="bottomMargin"><number>20</number></property>
      <property name="spacing"><number>12</number></property>

      <item>
       <widget class="QLabel" name="lblSpeedBig">
        <property name="text"><string>0</string></property>
        <property name="alignment"><set>Qt::AlignCenter</set></property>
        <property name="styleSheet"><string notr="true">QLabel{font-size:72px;font-weight:900;}</string></property>
       </widget>
      </item>

      <item>
       <widget class="QLabel" name="lblSpeedUnit">
        <property name="text"><string>km/h</string></property>
        <property name="alignment"><set>Qt::AlignCenter</set></property>
        <property name="styleSheet"><string notr="true">QLabel{color:#b8c0cc;font-size:18px;}</string></property>
       </widget>
      </item>

      <item>
       <layout class="QHBoxLayout" name="rowCards">
        <property name="spacing"><number>18</number></property>
        <item><widget class="QLabel" name="lblBattery"><property name="text"><string>100%</string></property><property name="styleSheet"><string notr="true">QLabel{font-size:18px;}</string></property></widget></item>
        <item><widget class="QLabel" name="lblGps"><property name="text"><string>GPS OK</string></property><property name="styleSheet"><string notr="true">QLabel{font-size:18px;}</string></property></widget></item>
        <item><widget class="QLabel" name="lblReverse"><property name="text"><string>D</string></property><property name="styleSheet"><string notr="true">QLabel{font-size:18px;font-weight:800;}</string></property></widget></item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="rowButtons">
        <property name="spacing"><number>12</number></property>
        <item><widget class="QPushButton" name="btnNav"><property name="text"><string>Navigation</string></property></widget></item>
        <item><widget class="QPushButton" name="btnCam"><property name="text"><string>Cam√©ra</string></property></widget></item>
       </layout>
      </item>

     </layout>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
